inst/extdata/load_evaporation.R

```diff
@@ -73,23 +73,22 @@ read_evaporation_matrix_from_url <- function(url)
 }
 
 # get geographical "stamp" for Berlin area ---------------------------------
-Berlin_DWD_mask <- function()
+berlin_dwd_mask <- function()
 {
   #DWD matrix filled with NA
-  DWD_dim <- matrix(NA, nrow = 866, ncol = 654)
-  
-  #get Berlin coordinates...I did not find out how to link to the file I uploaded!!!
-  Berlin_coordinates <- foreign::read.dbf('C:/Aendu_lokal/ABIMO_Paper/Daten/Karten/Hilfsgrid_DWD/Grid_Berlin_DWD.dbf')[,6:7]
-  
-  #set Berlin cells to 1 
-  Berlin_matrix <- DWD_dim
-  
-  for (i in 1:length(grid_Berlin_DWD$row)) {
-      Berlin_matrix[grid_Berlin_DWD$row[i], grid_Berlin_DWD$col[i]] <- 1
+  berlin_matrix <- matrix(NA, nrow = 866, ncol = 654)
+
+  #get Berlin coordinates
+  berlin_coordinates <- read.csv(system.file("extdata/berlin_coordinates.csv", package = "kwb.dwd"))
+
+  #set Berlin cells to 1
+
+  for (i in 1:length(berlin_coordinates$row)) {
```

Comment:
The expression `1:length(x)` may lead to problems if the length of `x` is zero. 

In this case the expression evaluates to `1:0` and finally to the decending sequence `c(1, 0)`. If this sequence appears in the header of a loop, the loop body is entered twice (once with the value `1` and the second time with the value `0`) instead of not being entered at all (which would be correct if there are no elements to loop through). 

You prevent errors by always using `seq_along(x)` instead of `1:length(x)`.
```suggestion
  for (i in seq_along(berlin_coordinates$row)) {
```